covid19_sfbayarea/data/sonoma.py

```diff
@@ -117,24 +117,83 @@ def transform_cases(cases_tag: element.Tag) -> Dict[str, TimeSeries]:
 
     return { 'cases': cases, 'deaths': deaths }
 
-def transform_transmission(transmission_tag: element.Tag) -> Dict[str, int]:
+def transform_transmission(transmission_tag: element.Tag, total_cases: int, normalize: bool = True) -> Dict[str, int]:
```

Comment:
These new arguments could use a brief description in the docstring (especially `normalize`).